tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/datatransfer/ImportTestUtils.java

```diff
@@ -82,27 +84,43 @@ static void resetCallCount() {
 			autoBuildCallCount.set(0);
 			fullBuildCallCount.set(0);
 			otherBuildTriggerTypes.clear();
+			instantiationCount.set(0);
 		}
 
 		static void assertAutoBuildWasDone() {
-			assertEquals("Expected no clean build triggers", 0, cleanBuildCallCount.get());
-			assertEquals("Expected 1 auto-build trigger", 1, autoBuildCallCount.get());
-			assertEquals("Expected no full build triggers", 0, fullBuildCallCount.get());
-			assertEquals("Expected only clean and auto-build triggers", Collections.EMPTY_LIST, otherBuildTriggerTypes);
+			assertBuildsWereDone(0, 1, 0);
 		}
 
 		static void assertFullBuildWasDone() {
-			assertEquals("Expected no clean build triggers", 0, cleanBuildCallCount.get());
-			assertEquals("Expected 1 full-build trigger", 1, fullBuildCallCount.get());
-			assertEquals("Expected no auto-build triggers", 0, autoBuildCallCount.get());
-			assertEquals("Expected only clean and auto-build triggers", Collections.EMPTY_LIST, otherBuildTriggerTypes);
+			assertBuildsWereDone(0, 0, 1);
 		}
 
 		static void assertNoBuildWasDone() {
-			assertEquals("Expected no clean build triggers", 0, cleanBuildCallCount.get());
-			assertEquals("Expected no full-build triggers", 0, fullBuildCallCount.get());
-			assertEquals("Expected no auto-build triggers", 0, autoBuildCallCount.get());
-			assertEquals("Expected no build triggers", Collections.EMPTY_LIST, otherBuildTriggerTypes);
+			assertBuildsWereDone(0, 0, 0);
+		}
+
+		private static void assertBuildsWereDone(int cleanBuilds, int autoBuilds, int fullBuilds) {
+			assertAll(
```

Comment:
I'd propose to use an existing `assertAll()` implementation, such as using AssertJ's `satisfies()` or its soft assertions. In contrast to a custom implementation, it will be easiert to maintain and provide proper error output.

Reply:
Thanks for the tip, I didn't know of that one :-)

Done in bd509baf7916fa80855670b7edafb47904360e35